bin/sdk/gutenberg.js

```diff
@@ -2,6 +2,8 @@
 /**
  * External dependencies
  */
+
+/* eslint import/no-nodejs-modules: ["error", {"allow": ["path", "fs"]}] */
```

Comment:
Noting that I made this rule global for `/bin` files in https://github.com/Automattic/wp-calypso/pull/28167 but no changes needed for now here.